build/constants.go

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
do we really need this file as contract ? if so, they should be built into the api.pb.go file.

Reply:
this file is at the same folder(package) of cri-api; better in pair with api.pb.go.